R/entity.R

```diff
@@ -40,6 +20,19 @@ Entity <- R6::R6Class(
     #' @field fullPath Same as \code{path}, but with the simulation name. (read-only)
     fullPath = function(value) {
       private$wrapExtensionMethod(EntityExtensions, "EntityPath", "fullPath", value)
+    },
+    #' @field parentContainer Returns a new wrapper instance to the .NET parent container. Multiple call to this method
+    #' will always return the same instance. However two children of the same parent will return two different instances of \code{Container}
+    #' pointing to the same .NET container
+    parentContainer = function(value) {

```

Comment:
Also parentContainer is now supported. Same .NET instance for all children. Different .net instance per child but always the same instance per child. So I think it's not bad